packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx

```diff
@@ -65,17 +65,15 @@ const CarePanel: React.FC<CarePanelProps> = ({ patientUuid, formEntrySub, launch
           </div>
         </CardHeader>
         <div style={{ width: '100%', minHeight: '20rem' }}>
-          {switchItem?.name === 'HIV Program' ? (
-            <>
-              <ProgramEnrollment
-                patientUuid={patientUuid}
-                programName={switchItem?.name}
-                enrollments={enrollments}
-                formEntrySub={formEntrySub}
-                launchPatientWorkspace={launchPatientWorkspace}
-                PatientChartProps={''}
-              />
-            </>
+          {programEnrolled === 'HIV Program' ? (
```

Comment:
Probably add something for the other programs is there's no data on them.. `No data for selected program`